libc/src/__support/RPC/rpc.h

```diff
@@ -57,7 +57,7 @@ template <uint32_t lane_size = gpu::LANE_SIZE> struct alignas(64) Packet {
 };
 
 /// The maximum number of parallel ports that the RPC interface can support.
-constexpr uint64_t MAX_PORT_COUNT = 512;
+constexpr uint64_t MAX_PORT_COUNT = 4096;
```

Comment:
This constant is spread across multiple files.  Do you want to have a macro defining this something like `LIBC_RPC_MAX_PORT_COUNT` defaulted to 4096 if undef, so that it can even be set from the build command?

Reply:
I'm unsure if it's worthwhile to have such a thing. The only reason we define this in multiple places is because the original discussions with Siva indicated that we didn't want to simply provide access to the `rpc.h` header. So, the compromise was to define it twice and have a static assert in the impl to ensure they always match. I just can't imagine a situation where a user would want to set this manually (maybe save 128 bytes of GPU memory?) so it's probably best not to bloat the options.